### PR TITLE
Feature to improve stats app charts

### PR DESCRIPTION
**Pull request by forbesus**
refs #298

### CODE REVIEW

**Comment by forbesus** [comment]
hello @baktun14 
I have done #298 would you check this pr?

**Comment by baktun14** [comment]
Hey @forbesus I just tested and it works great! I have a few modifications requirements:
- Would it possible to remove the TV logo?
- When you switch between 7d 1M ALL, the zoom works but it only includes the data of that range so when you zoom out in 7d/1M, it doesn't really work. There's some logic that slices the data coming from props `rangedData`
- Would it possible to have less padding by default at the top and bottom of the chart?
image
- Would it be possible to have the same unit as the current value in the right axis labels? 
image
image
- Would it be possible to decrease the line width a tiny bit?

Thanks for your contribution!

File: apps/stats-web/package.json [hunk 1]
```diff
@@ -24,7 +24,9 @@
     "geist": "^1.3.0",
     "iconoir-react": "^7.3.0",
     "jotai": "^2.5.1",
+    "lightweight-charts": "^4.2.0",
     "lucide-react": "^0.292.0",
+    "moment": "^2.30.1",
```
Comment (baktun14) [inline apps/stats-web/package.json]: Please remove moment and use `data-fns`, that's the library we use in other projects for time based functions.

File: apps/stats-web/src/components/graph/Graph.tsx [hunk 2]
```diff
@@ -25,74 +30,140 @@ const Graph: React.FunctionComponent<IGraphProps> = ({ rangedData, snapshotMetad
...
-              x: _snapshot.date,
-              y: roundDecimal(snapshotMetadata.unitFn(_snapshot.value).value)
-            }))
-            .sort(function (a, b) {
-              return Number(new Date(a.x)) - Number(new Date(b.x));
-            })
+    ? rangedData.map(_snapshot => (
+      {
+        time: moment.utc(_snapshot.date).format('YYYY-MM-DD'),
+        value: _snapshot.value
```
Comment (baktun14) [inline apps/stats-web/src/components/graph/Graph.tsx]: There was a unitFn called in the previous code `y: roundDecimal(snapshotMetadata.unitFn(_snapshot.value).value)` which is probably why the values don't match.

**Review by baktun14 — CHANGES_REQUESTED** [review]

**Comment by forbesus** [comment]
Hello @baktun14 i fixed and completed
Would you check again?

**Comment by baktun14** [comment]
@forbesus Great, I will check today.

**Comment by baktun14** [comment]
@forbesus  There's a bug when changing the time range, it duplicates the graph. I'm investigating it.
image

**Comment by forbesus** [comment]
Hello @baktun14 I checked it but I have a question
Did you check it after removing browser cache?

**Comment by baktun14** [comment]
> Hello @baktun14 I checked it but I have a question Did you check it after removing browser cache?

It doesn't change the behavior for me

**Comment by forbesus** [comment]
Hello @baktun14 
I fixed that problem 
Would you check again?

**Comment by baktun14** [comment]
@forbesus It works great!

**Review by baktun14 — APPROVED** [review]

**Comment by baktun14** [comment]
Great work @forbesus ! I fixed a resize issue and theme issue, but otherwise all is good to merge.

**Comment by forbesus** [comment]
Thank you @baktun14

**Pull request merged**